src/py42/modules/archive.py

```diff
@@ -1,32 +1,108 @@
 
     def get_backup_sets(self, device_guid, destination_guid):
+        """Gets all backup set names/identifiers referring to single destination for a specific
+        device.
+        `Support Page <https://support.code42.com/Administrator/Cloud/Configuring/Backup_sets>`__
+
+        Args:
+            device_guid (str): The GUID for the device to get backup sets for.
+            destination_guid (str): The GUID for the destination containing the archive to get
+                backup sets for.
+
+        Returns:
+            :class:`py42.sdk.response.Py42Response`: A response containing the backup sets.
+        """
         return self._archive_client.get_backup_sets(device_guid, destination_guid)
 
-    def get_data_key_token(self, device_guid):
-        return self._archive_client.get_data_key_token(device_guid)
+    def get_all_restore_history_by_org_id(self, days, org_id):
+        """Gets all restore jobs from the past given days for the organization with the given ID.
+        `REST Documentation <https://console.us.code42.com/apidocviewer/#RestoreHistory-get>`__
 
-    def get_restore_history_by_org_id(self, days, org_id, page_num=None, page_size=None):
-        return self._archive_client.get_restore_history(days, u"orgId", org_id, page_num, page_size)
+        Args:
+            days (int): Number of days of restore history to retrieve.
+            org_id (int): The ID for the organization to get restore history for.
 
-    def get_restore_history_by_user_id(self, days, user_id, page_num=None, page_size=None):
-        return self._archive_client.get_restore_history(
-            days, u"userId", user_id, page_num, page_size
-        )
+        Returns:
+            generator: An object that iterates over :class:`py42.sdk.response.Py42Response` objects
+            that each contain a page of restore history.
+        """
+        return self._archive_client.get_all_restore_history(days, u"orgId", org_id)
 
-    def get_restore_history_by_device_id(self, days, device_id, page_num=None, page_size=None):
-        return self._archive_client.get_restore_history(
-            days, u"computerId", device_id, page_num, page_size
-        )
+    def get_all_restore_history_by_user_id(self, days, user_id):
+        """Gets all restore jobs from the past given days for the user with the given ID.
+        `REST Documentation <https://console.us.code42.com/apidocviewer/#RestoreHistory-get>`__
+
+        Args:
+            days (int): Number of days of restore history to retrieve.
+            user_id (int): The ID for the user to get restore history for.
+
+        Returns:
+            generator: An object that iterates over :class:`py42.sdk.response.Py42Response` objects
+            that each contain a page of restore history.
+        """
+        return self._archive_client.get_all_restore_history(days, u"userId", user_id)
+
+    def get_all_restore_history_by_device_id(self, days, device_id):
```

Comment:
Can we rename these to be in this format:
`get_all_device_restrore_history`
`get_all_user_restore_history`
`get_all_org_restore_history`

the names are a little more concise

Reply:
Also added tests for these methods